M ◾ Several improvements to google-my-business-profile (#9961)
SSWConsulting/SSW.Rules.Content: rules/why-your-business-should-be-on-google-my-business/rule.md

  - title: Camilla Rosa Silva
    url: https://www.ssw.com.au/people/camilla-rosa-silva
created: 2022-05-10T23:09:37.336Z
related:
  - optimize-google-my-business-profile
guid: e92c71d8-38a9-4d30-b363-78841fc9a538

---

Google My Business is a business listing platform provided by Google. Business owners can create or claim a previously created listing of their business.
The goal of a Google My Business (GMB) profile is to have your business known by local users (caused by brand awareness), and ultimately, increase your sales.

<!--endintro-->

By adding your business information to Google, you will:

* Have your business show up on Google Maps
* Provide important information to searchers (aka potential customers)
* Receive business reviews and build trust with your clientele

> If don’t have a Google My Business profile yet, it’s unlikely your business will show up on Google Maps and be searchable by Google users. A GMB helps ensure that people find your business when looking for products and services like yours in their local area.

Reviewer: ```suggestion
If don’t have a Google My Business profile yet, it’s unlikely your business will show up on Google Maps and be searchable by Google users. A GMB profile helps ensure that people find your business when looking for products and services like yours in their local area.
```

or:

```suggestion
If don’t have a Google My Business profile yet, it’s unlikely your business will show up on Google Maps and be searchable by Google users. GMB helps ensure that people find your business when looking for products and services like yours in their local area.
```

Note that if you have any poms they may relate GMB to Good Morning Britain (a breakfast show in the UK)

Let’s imagine a customer is trying to find your business specifically, or a similar business around their location. The customer may not be able to find you without a Google profile/listing.

> Now, let’s think of a scenario where a listing for your business already exists, but you don’t have a profile and therefore, have no control over the information in the existent profile for your business; or worse, the information is wrong, but you can’t fix it. This is why it’s important to have a profile on GMB, or for you to claim your GMB profile and [manage it yourself](/optimize-google-my-business-profile).

Reviewer: ```suggestion
Now, imagine a scenario where a listing for your business already exists, but you don’t have a profile, and therefore have no control over the information in the listing. Or worse, the information is wrong and you can’t fix it. This is why it’s critical to create a profile on GMB, or claim your business listing in your GMB profile so you can [manage it yourself](/optimize-google-my-business-profile).
```
